content/en/cosign/overview.md

```diff
@@ -15,98 +15,84 @@ features:
-### 1. Generate a keypair
 
-```shell
-$ cosign generate-key-pair
-Enter password for private key:
-Enter again:
-Private key written to cosign.key
-Public key written to cosign.pub
 ```
-
-### 2. Sign a container and store the signature in the registry
-
-```shell
-$ cosign sign --key cosign.key user/demo
-Enter password for private key:
-Pushing signature to: index.docker.io/user/demo:sha256-87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8.sig
+$ cosign sign-blob <file> --bundle txtbundle.bundle
 ```
 
-The `cosign` command above prompts the user to enter the password for the private key.
-The user can manually enter the password, or set an environment variable with `COSIGN_PASSWORD` to use a password automatically.
-
-### 3. Verify a container against a public key
+The bundle contains signing metadata, including the signature and certificate.  
 
-This command returns `0` if *at least one* `cosign` formatted signature for the image is found
-matching the public key. Review the other sections of this site for information and caveats on other signature formats.
+The Cosign command requests a certificate from our Certificate Authority Fulcio. Fulcio checks your identity by using an authentication protocol (OpenID Connect) to look at your email address. If your identity is correct, Fulcio grants a short-lived, time-stamped certificate. The certificate is bound to the public key to attest to your identity.  This activity is logged using Rekor.
+ 
+Note that you don’t need to use a key to sign.  Currently, you can authenticate with Google, GitHub, or Microsoft. For more information, read [Keyless Signatures](https://docs.sigstore.dev/cosign/keyless/).
 
-Any valid payloads are printed to `stdout`, in JSON format.
-Note that these signed payloads include the digest of the container image, which is how we can be
-sure these "detached" signatures cover the correct image.
+Cosign has additional options and features.   For more information enter the command:
 
-```shell
-$ cosign verify --key cosign.pub dlorenc/demo
-The following checks were performed on these signatures:
-  - The cosign claims were validated
-  - The signatures were verified against the specified public key
-{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8"},"Type":"cosign container image signature"},"Optional":null}
+```
+cosign sign-blob --help
 ```
 
-## Kubernetes Integrations
+### Verifying a signed blob
 
-Cosign comes with a few built-in Kubernetes integrations: `Secret` generation, and a [policy webhook](/policy-controller/overview/) `policy-controller`.
-In addition to the `policy-controller`, Cosign is also compatible with and supported by other policy engines such as:
+To verify a signed blob, you need to provide three pieces of information:
+* The certificate, found in the bundle
+* The signature, also found in the bundle
+* The identity used in signing
```

Comment:
Will users know where/how to find the identity?

Reply:
We don't want to tell them to just look at the certificate. Maybe we can say something like "The blob maintainer should provide their trusted identity"

Reply:
Is this an important enough point to specify "The blog maintainer should provide the trusted identity separately from the certificate"?

Reply:
What do we want to do here? This seems to be the last outstanding item @haydentherapper @jonvnadelberg

Reply:
Kara's text LGTM